src/iop/watermark.c

```diff
@@ -311,8 +370,73 @@ static gchar *_watermark_get_svgdoc(dt_iop_module_t *self, dt_iop_watermark_data
       svgdata = svgdoc;
     }
 
-    // Current image ID
+    // Simple text from watermark module
     gchar buffer[1024];
+
+    if (p->font)
```

Comment:
Wouldn't have happened if you didn't merged unready branch. Again.
And i call it "unready" because no-one (not even i, after that review, which wasn't final/finished), have said "can not find any more issues there, looks good to merge".
Not cool.